src/jdk.incubator.jextract/share/classes/jdk/internal/clang/LibClang.java

```diff
@@ -49,12 +48,14 @@
             //this is an hack - needed because clang_toggleCrashRecovery only takes effect _after_ the
             //first call to createIndex.
```

Comment:
Maybe add a link to:
https://reviews.llvm.org/D23662

This should really get fixed upstream.